SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -220,13 +227,20 @@ LAYEREDLBSTRATEGY_TEST(GIVEN_strat_is_not_ready_WHEN_local_virtual_worker_id_is_
 	ADD_LATENT_AUTOMATION_COMMAND(FCheckStratIsReady(Data, this, false));
 	ADD_LATENT_AUTOMATION_COMMAND(FSetLocalVirtualWorker(Data, 1));
 	ADD_LATENT_AUTOMATION_COMMAND(FCheckStratIsReady(Data, this, true));
+	ADD_LATENT_AUTOMATION_COMMAND(FCleanup(Data));
+
+	Settings->bAutoStartLocalDeployment = OldBAutoStartLocalDeployment;
 
 	return true;
 }
 
 LAYEREDLBSTRATEGY_TEST(
 	GIVEN_layered_strat_of_two_by_four_grid_strat_singleton_strat_and_default_strat_WHEN_get_minimum_required_workers_called_THEN_ten_returned)
 {
+	const auto Settings = GetMutableDefault<USpatialGDKEditorSettings>();
```

Comment:
This would be a good candidate for a test fixture (or some other form of refactoring).
Have a look at `SchemaTestFixture`, which is used for our schema tests.
I think we could bundle the setting changes into it at least, maybe even the AutomationOpenMap and FCleanup.

Reply:
Reordering the statements as you mentioned in a different comment makes this pretty simple to refactor into a `SetUp` method :)